adagrams/game.py

```diff
@@ -1,11 +1,51 @@
+import random
+from adagrams.const_dicts import *
+
 def draw_letters():
-    pass
+    letter_pool_list = []
+    for letter, number in LETTER_POOL.items():
+        letter_pool_list.extend([letter]*number)
+    hand_list = []
+    while len(hand_list) < 10:
+        random.shuffle(letter_pool_list)
+        hand_list.append(letter_pool_list.pop())
```

Comment:
This works well to generate a list of 10 random letters, but it's doing a lot of extra work.  `random.shuffle` will randomize the list, and once it's randomized, selecting the first 10 letters will give you a random hand of letters.  Reshuffling between each selection isn't necessary.